src/project/mod.rs

```diff
@@ -955,6 +956,21 @@ impl Project {
 
         Ok(implicit_constraints)
     }
+
+    /// Returns true if all preview features are enabled
+    pub fn all_preview_features_enabled(&self) -> bool {
+        self.manifest
+            .preview()
+            .map(|preview| preview.all_enabled())
+            .unwrap_or(false)
+    }
+
+    pub fn is_preview_feature_enabled(&self, feature: &KnownPreviewFeature) -> bool {
```

Comment:
I assume `KnownPreviewFeature` is a `Copy` type so there is no need to pass it by reference!